src/javascripts/components/copy.js

```diff
@@ -1,26 +1,41 @@
-import $ from 'jquery'
+import 'govuk-frontend/vendor/polyfills/Event'
 import ClipboardJS from 'clipboard'
 
-var Copy = {
-  init: function (selector) {
-    $(selector).parent().prepend('<button class="app-copy-button js-copy-button" aria-live="assertive">Copy</button>')
-    // Copy to clipboard
-    try {
-      new ClipboardJS('.js-copy-button', {
-        target: function (trigger) {
-          return trigger.nextElementSibling
-        }
-      }).on('success', function (e) {
-        e.trigger.textContent = 'Copied'
-        e.clearSelection()
-        setTimeout(function () {
-          e.trigger.textContent = 'Copy'
-        }, 5000)
-      })
-    } catch (err) {
-      if (err) {
-        console.log(err.message)
+function Copy ($module) {
+  this.$module = $module
+}
+
+Copy.prototype.init = function () {
+  var $module = this.$module
+  if (!$module) {
+    return
+  }
+  var $button = document.createElement('button')
+  $button.className = 'app-copy-button js-copy-button'
+  $button.setAttribute('aria-live', 'assertive')
+  $button.textContent = 'Copy'
+
+  var $parent = $module.parentNode
+  $parent.insertBefore($button, $parent.firstChild)
```

Comment:
Heads up for anyone else reviewing that https://github.com/alphagov/govuk-design-system/pull/518 will remove the need to look for the parent element here which is good as it feels a bit brittle (not the fault of this PR, this replicates the existing functionality).